pkg/config/config_template.yaml

```diff
@@ -306,6 +306,30 @@ metadata_collectors:
 #
 {{ end -}}
 
+# Datadog Cluster Agent - DCA
+#
+# The Authentication Token used to query the Datadog cluster agent from the node agent.
+# This token is provided in the header "Authorization" of each request, its length be must
```

Comment:
`its length be must` -> `its length must be`

also, not sure we should mention that it's in the "Authorization" header, to me it's more of an implementation detail/should be documented somewhere else